### PR TITLE
Document the URLField OpenAPI issue

### PR DESCRIPTION
**Pull request by fao89**
https://pulp.plan.io/issues/6828
closes #6828

Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/contributing/pull-request-walkthrough.html

### CODE REVIEW

**Review by bmbouter — APPROVED** [review]
This looks great. Really clear, good writing.

File: docs/plugins/reference/how-plugins-work.rst [hunk 1]
```diff
@@ -72,3 +72,14 @@ components, particularly for both models and viewsets.
 
 Take a look at `the structure <https://github.com/pulp/pulp_file/tree/master/pulp_file/app>`_ of
 the ``pulp_file`` plugin.
+
+
+Serializer and OpenAPI schema
+-----------------------------
+
+The serializers are converted to OpenAPI objects through `drf-spectacular <https://github.com/tfranzel/drf-spectacular>`_.
```
Comment (ipanova) [inline docs/plugins/reference/how-plugins-work.rst:80]: s/to OpenAPI objects/to the OpenAPI objects

File: docs/plugins/reference/how-plugins-work.rst [hunk 2]
```diff
@@ -72,3 +72,14 @@ components, particularly for both models and viewsets.
...
+It inspects all serializer fields to describe them in the OpenAPI schema.
+Due to a `DRF issue <https://github.com/encode/django-rest-framework/issues/7354>`_
```
Comment (ipanova) [inline docs/plugins/reference/how-plugins-work.rst:82]: s/Due to a/Due to the

**Review by ipanova — APPROVED** [review]

File: docs/plugins/reference/how-plugins-work.rst (hunk 1 above)
Comment (daviddavis) [reply docs/plugins/reference/how-plugins-work.rst:80]: Actually, this should probably be "Serializers are converted to OpenAPI objects through...".

**Pull request merged**